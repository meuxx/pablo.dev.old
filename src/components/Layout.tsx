import React from 'react'
import { library, config } from '@fortawesome/fontawesome-svg-core'
import '@fortawesome/fontawesome-svg-core/styles.css'
import { fab } from '@fortawesome/free-brands-svg-icons'
import { faEnvelope } from '@fortawesome/free-regular-svg-icons'
import GlobalStyle from '../styles/GlobalStyle'
import Header from './Header'

config.autoAddCss = false
library.add(fab, faEnvelope)

const Layout: React.FC = ({ children }) => (
  <div>
    <GlobalStyle />
    <Header />
    {children}
  </div>
)

export default Layout
