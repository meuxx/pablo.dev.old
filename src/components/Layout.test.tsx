import React from 'react'
import { render } from '@testing-library/react'
import Layout from './Layout'

jest.mock('./Header', () => () => <div />)

describe('Layout', () => {
  it('renders without crashing', () => {
    const { container } = render(<Layout />)

    expect(container).toBeDefined()
    expect(container).toMatchSnapshot()
  })
})
