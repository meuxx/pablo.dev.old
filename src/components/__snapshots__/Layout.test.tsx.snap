// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders without crashing 1`] = `
<div>
  <div>
    <div />
  </div>
</div>
`;
